@here/harp-mapview/lib/DecodedTileHelpers.ts

```diff
@@ -101,80 +101,88 @@ export function createMaterial(
     material.depthTest = isExtrudedPolygonTechnique(technique) && technique.depthTest !== false;
 
     if (isStandardTexturedTechnique(technique)) {
-        ["map", "normalMap", "displacementMap"].forEach((texturePropertyName: string) => {
-            const textureProperty = (technique as any)[texturePropertyName];
-            if (textureProperty === undefined) {
-                return;
-            }
+        ["map", "normalMap", "displacementMap", "roughnessMap"].forEach(
```

Comment:
Since the aim of this ticket is to explore three's ``MeshStandardMaterial``, why don't we expose all the other maps that can come into play in the PBR light model?
* Ao
* Metalness
* EnvMap

You can find all of them here: https://threejs.org/docs/index.html#api/en/materials/MeshStandardMaterial

Reply:
good point, but at least aoMap, lightMap need additional set of texture coordinates. Implementing that would exceed the scope of this task. Will add the others though.